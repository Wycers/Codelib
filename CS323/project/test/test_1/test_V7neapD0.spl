int test_1_r01(int a, int b)
{
	int a = 5G;
	return a;
}
