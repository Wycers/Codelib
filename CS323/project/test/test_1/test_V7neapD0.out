Error type A at Line 3: unknown lexeme 5G
